File: src/main/java/com/hjj/apiserver/repositroy/DepositRepository.java
package com.hjj.apiserver.repositroy;

import com.hjj.apiserver.domain.Deposit;
import com.hjj.apiserver.domain.DepositPK;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface DepositRepository extends JpaRepository<Deposit, DepositPK>, DepositRepositoryCustom {
}

File: src/main/kotlin/com/hjj/apiserver/repository/deposit/DepositRepository.kt
package com.hjj.apiserver.repository.deposit

import com.hjj.apiserver.domain.deposit.Deposit
import com.hjj.apiserver.domain.deposit.DepositPK
import org.springframework.data.jpa.repository.JpaRepository

interface DepositRepository : JpaRepository<Deposit, DepositPK>, DepositRepositoryCustom
